app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java

```diff
@@ -0,0 +1,28 @@
+package com.poupa.vinylmusicplayer.helper.menu;
+
+
+import android.content.Context;
+import android.text.SpannableString;
+import android.text.style.ForegroundColorSpan;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import com.poupa.vinylmusicplayer.R;
+
+
+public class MenuHelper {
+
+    public static void setDeleteMenuItemRed(Menu menu, Context context) {
+        // All delete element inside of menu should have red text to better differentiate them
+        MenuItem liveItem = menu.findItem(R.id.action_delete_playlist);
+        if (liveItem == null)
+            liveItem = menu.findItem(R.id.action_delete_from_device);
+
+        if (liveItem != null) {
+            SpannableString s = new SpannableString(liveItem.getTitle().toString());
+            int color = context.getResources().getColor(R.color.delete);
```

Comment:
Looks like this call is deprecated: https://stackoverflow.com/a/31590927

Reply:
will change this

Reply:
done